source/view/preview.cpp

```diff
@@ -1,17 +1,20 @@
 #include "preview.hpp"
+#include <ftxui/component/event.hpp>
 #include <sstream>
 
 namespace caps_log::view {
 using namespace ftxui;
 
-Element Preview::Render() {
+Element Preview::OnRender() {
     Elements visibleLines;
+    visibleLines.push_back(m_title);
     for (int i = m_topLineIndex; i < m_lines.size(); i++) {
         visibleLines.push_back(m_lines[i]);
     }
 
     constexpr auto kHeight = 14;
-    auto element = window(m_title, vbox(visibleLines)) | flex_shrink | size(HEIGHT, EQUAL, kHeight);
+    auto element = vbox(visibleLines) | borderRounded | size(HEIGHT, EQUAL, kHeight);
```

Comment:
It's difficult to set the title to be centred for a `window` (https://github.com/ArthurSonzogni/FTXUI/issues/1016) . Using `vbox` instead. Not as pretty but it will do.
- [ ]  link issue in the code comment